src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -79,7 +102,7 @@ public class CommandTestUtil {
      * - the {@code actualModel} matches {@code expectedModel}
      */
     public static void assertCommandSuccess(Command command, Model actualModel, CommandResult expectedCommandResult,
-            Model expectedModel) {
+                                            Model expectedModel) {
```

Comment:
Oh, I'm not sure whether this is the correct indentation level? I think the old one is correct

Reply:
Got it, will fix the checkstyles one shot!!